org.unmojang.FjordLauncher.yml

```diff
@@ -0,0 +1,167 @@
+id: org.unmojang.FjordLauncher
```

Comment:
Getting a SSL error on the domain

![image](https://github.com/flathub/flathub/assets/62639087/8ac3023c-2e8c-443e-906e-3b85447eba8b)

Reply:
Yes, there's nothing there yet, there is a website at https://drasl.unmojang.org however. I can put up a placeholder page on unmojang.org if needed.

Reply:
> I can put up a placeholder page on unmojang.org if needed.

Yes please do

Reply:
Done: https://unmojang.org/